### ISSUE TITLE
About the size of  the input test data

### ISSUE THREAD

**Issue by azyslzp** [IsaacCorley/pytorch-modulation-recognition#3]
Hi, Thanks a lot for your sharing! The code have trained and tested in my Server, and there still has a question as, 
due to the input size of the vtcnn and mrresnet is fixed to adapt the training data, then if I want to test by using the signal data generated or collected by us, which the length is not the same as the trainning data, how should we do?
cut or sample it?
Thanks a lot~

**Comment by isaaccorley** [comment]
@azyslzp The dataset used for training is the RML2016.10b dataset used in DeepSig's paper [Convolutional Radio Modulation Recognition Networks](https://arxiv.org/abs/1602.04105). In the paper it states that each window is composed of 128 microsecond samples implying a 1 MHz sampling frequency and contains between 8-16 symbols. Intuitively, I would think resampling to this frequency while keeping enough symbols in the window based on your test set's baud rate would be appropriate. A side note: the MRResNet model utilizes a [Global Average Pooling](https://discuss.pytorch.org/t/how-can-i-perform-global-average-pooling-before-the-last-fully-connected-layer/74352) before the linear layers which means the model isn't constrained to a fixed size input, however you will likely see a performance dip depending on how different your test data is to the training set. Hope this helps.
